store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -234,12 +234,18 @@ protected void addAdditionalOperationHandlers() {
         try {
             addOperationHandler(AddElementsFromHdfs.class, new AddElementsFromHdfsHandler());
             addOperationHandler(GetElementsBetweenSets.class, new GetElementsBetweenSetsHandler());
-            addOperationHandler(GetElementsInRanges.class, new GetElementsInRangesHandler());
             addOperationHandler(GetElementsWithinSet.class, new GetElementsWithinSetHandler());
             addOperationHandler(SplitTable.class, new SplitTableHandler());
             addOperationHandler(SampleDataForSplitPoints.class, new SampleDataForSplitPointsHandler());
             addOperationHandler(ImportAccumuloKeyValueFiles.class, new ImportAccumuloKeyValueFilesHandler());
-            addOperationHandler(SummariseGroupOverRanges.class, new SummariseGroupOverRangesHandler());
+
+            if (null == getSchema().getVertexSerialiser() || getSchema().getVertexSerialiser().preservesObjectOrdering()) {
+                addOperationHandler(SummariseGroupOverRanges.class, new SummariseGroupOverRangesHandler());
+                addOperationHandler(GetElementsInRanges.class, new GetElementsInRangesHandler());
+            } else {
+                LOGGER.warn("Accumulo range scan operations will not be available on this store as the vertex serialiser does not preserve object ordering. Vertex serialiser: {}",
+                        null != getSchema().getVertexSerialiser() ? getSchema().getVertexSerialiser().getClass().getName() : " null.");
```

Comment:
Is checking Null pointless for the Logger message?
`If getSchema().getVertexSerialiser() == null` it will never reach the else condition, because it will satisfy the first condition of the OR expression.